<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrefsDataStoreWidget</class>
 <widget class="QWidget" name="PrefsDataStoreWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>327</height>
   </rect>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>9</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="1" column="0">
    <widget class="QGroupBox" name="mainpathGB">
     <property name="title">
      <string>Main Path</string>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>9</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="1">
       <widget class="QPushButton" name="mainSelBtn">
        <property name="toolTip">
         <string>Background color of the NULL value cell</string>
        </property>
        <property name="text">
         <string>Select..</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLineEdit" name="mainPath">
        <property name="toolTip">
         <string>Text to be used as a NULL mark</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="indexSavePathGB">
     <property name="title">
      <string>Index Save Path</string>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>9</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="1">
       <widget class="QPushButton" name="indexSelBtn">
        <property name="toolTip">
         <string>Background color of the BLOB value cell</string>
        </property>
        <property name="text">
         <string>Select..</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLineEdit" name="indexPath">
        <property name="toolTip">
         <string>Text to be used as a BLOB mark</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Data Store</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="backUpPathGB">
     <property name="title">
      <string>BackUp Path</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLineEdit" name="backPath"/>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="backSelBtn">
        <property name="text">
         <string>Select..</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QGroupBox" name="exportDefaultPathGB">
     <property name="title">
      <string>Export Default Path</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLineEdit" name="exportPath"/>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="exportSelBtn">
        <property name="text">
         <string>Select..</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
